app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MoodEvent.java

```diff
@@ -0,0 +1,156 @@
+package io.github.cmput301f19t19.legendary_fiesta;
+
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.Date;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Represents a single mood event
+ */
+public class MoodEvent {
+
+    /**
+     * An enum representing the possible social conditions that a MoodEvent occurred in
+     */
+    public enum SocialCondition {
+        SINGLE,
+        PAIR,
+        SMALL_GROUP,
+        CROWD
+    }
+
+    private Mood mood;
+    private String user;
+    private String description;
+    private Date date;
+    private SocialCondition condition;
+    private byte[] photo;
+    private LatLng location;
+
+    /**
+     * Constructor for mood event
+     * @param mood Required mood
+     * @param user Required user
+     * @param description Optional description
+     * @param date Required date
+     * @param condition Optional social condition
+     * @param photo Optional photo byte array
+     * @param location Optional location
+     */
+    public MoodEvent(@Nonnull Mood mood, @Nonnull String user, @Nullable String description,
+                     @Nonnull Date date, @Nullable SocialCondition condition,
+                     @Nullable byte[] photo, @ Nullable LatLng location) {
```

Comment:
Nit: There is an extra whitespace
```suggestion
                     @Nullable byte[] photo, @Nullable LatLng location) {
```